packages/wonder-blocks-dropdown/package.json

```diff
@@ -40,6 +38,7 @@
     "react-window": "^1.8.5"
   },
   "devDependencies": {
+    "@khanacademy/wonder-blocks-button": "^3.0.3",
```

Comment:
Discovered that this package is only used in SB stories.

I have a separate ticket to check for inter-dependencies that I'll be working on next sprint.